src/entrypoint/git-version.cr

```diff
@@ -0,0 +1,24 @@
+require "option_parser"
+
+require "file_utils"
+
+require "../git-version"
+
+dev_branch = "dev"
+folder = FileUtils.pwd
+
+OptionParser.parse! do |parser|
+  parser.banner = "Usage: git-version [arguments]"
+  parser.on("-folder FOLDER", "--folder=FOLDER", "Execute the command in the defined folder") { |f| folder = f }
```

Comment:
It might be convenient to have short versions of these flags, for instance `-f` for folder and `-b` for branch

Reply:
done